leetcodePython/SlidingWindow/slidingwindow_2747.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+from collections import Counter
+
+class Solution:
+    def countServers(self, n: int, logs: List[List[int]], x: int, queries: List[int]) -> List[int]:
+        result = [0] * len(queries)
+
+        query_map = sorted((time, index) for time, index in enumerate(queries))
+
+        # sort the logs base on request time
+        logs.sort(key=lambda l: l[1])
+        left = right = 0
+        server_usage = Counter()
+        available_server = n
```

Comment:
It's possible a server used more than 1 time in anytime frame. So we need to count each server usage frequency (or usage count)